server/Controllers/ideaController.js

```diff
@@ -5,117 +5,142 @@ const ideaController = {};
 
 // middleware to get all ideas data from database
 ideaController.getIdeas = (req, res, next) => {
-	const queryText = 'SELECT * FROM Ideas';
+  const queryText = 'SELECT * FROM Ideas';
 
-	model.query(queryText, (err, results) => {
-		if (err) {
-			console.log(err);
-			return next({
-				log: `error occurred at getIdeas middleware. error message is: ${err}`,
-				status: 400,
-				message: { err: 'An error occurred' },
-			});
-		}
-		// console.log('results', results.rows);
-		res.locals.ideas = results.rows;
-		return next();
-	});
+  model.query(queryText, (err, results) => {
+    if (err) {
+      console.log(err);
+      return next({
+        log: `error occurred at getIdeas middleware. error message is: ${err}`,
+        status: 400,
+        message: { err: 'An error occurred' },
+      });
+    }
+    // console.log('results', results.rows);
+    res.locals.ideas = results.rows;
+    return next();
+  });
 };
 
 // INSERT INTO Ideas (name, description, why, when_start, when_end, who, image, creator_username) VALUES ('scratch', 'scratch project', 'for fun', '2020-07-25', '2020-08-15', '3', 'image.png', 'hello1');
 
 // we need to know who's submitting the idea
 ideaController.submitIdea = (req, res, next) => {
-	const {
-		name,
-		description,
-		why,
-		whenStart,
-		whenEnd,
-		teamNumber,
-		imageURL,
-		username,
-	} = req.body;
+  const {
+    name,
+    description,
+    why,
+    techStack,
```

Comment:
Ok I see you're just accommodating the new functionality of a User submitting their idea with a tech stack.